gui.go

```diff
@@ -165,6 +169,10 @@ func (g *Gui) Size() (x, y int) {
 	return g.maxX, g.maxY
 }
 
+func (g *Gui) MousePosition() (x, y int) {
```

Comment:
Can you add documentation to this method?

Reply:
Done :)